instrumentation/elasticsearch/elasticsearch-transport-2.0/elasticsearch-transport-2.0.gradle

```diff
@@ -1,46 +0,0 @@
-// Set properties before any plugins get loaded
-ext {
-  minJavaVersionForTests = JavaVersion.VERSION_1_8
-  maxJavaVersionForTests = JavaVersion.VERSION_1_8
-}
-
-apply from: "$rootDir/gradle/instrumentation.gradle"
-apply plugin: 'org.unbroken-dome.test-sets'
-
-muzzle {
-  pass {
-    group = "org.elasticsearch"
-    module = "elasticsearch"
-    versions = "[2.0,3)"
-    assertInverse = true
-  }
-}
-
-testSets {
-  latestDepTest
-}
-
-dependencies {
-  compileOnly group: 'org.elasticsearch', name: 'elasticsearch', version: '2.0.0'
-
-  implementation project(':instrumentation:elasticsearch:elasticsearch-transport-common')
-
-  // Ensure no cross interference
-  testImplementation project(':instrumentation:elasticsearch:elasticsearch-rest-5.0')
-  testImplementation project(':instrumentation:elasticsearch:elasticsearch-transport-5.0')
-  testImplementation project(':instrumentation:apache-httpasyncclient-4.0')
-
-  testImplementation group: 'org.elasticsearch', name: 'elasticsearch', version: '2.0.0'
-
-  testImplementation group: 'org.springframework.data', name: 'spring-data-elasticsearch', version: '2.0.0.RELEASE'
-
-  testImplementation group: 'net.java.dev.jna', name: 'jna', version: '4.5.1'
-  testImplementation group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.11.0'
-  testImplementation group: 'org.apache.logging.log4j', name: 'log4j-api', version: '2.11.0'
-
-  testImplementation group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-smile', version: '2.9.9'
-  // ^ is needed because we are using a newer version of jackson that isn't compatible without this.
-
-  latestDepTestImplementation group: 'org.elasticsearch', name: 'elasticsearch', version: '2.4.6'
-  latestDepTestImplementation group: 'org.springframework.data', name: 'spring-data-elasticsearch', version: '2.1.15.RELEASE'
```

Comment:
If not removing this instrumentation, I would probably rename this testset since it doesn't follow our convention of using a wildcard version.